.npmignore

```diff
@@ -6,7 +6,6 @@
 /tmp
 /fastboot-dist
 /fastboot-tests
-/docs
```

Comment:
We actually need these files so we can run a production build on now which automatically builds the docs.